services/QuillLMS/client/app/bundles/Shared/utils/constants.ts

```diff
@@ -44,6 +44,9 @@ export const EVIDENCE_HANDBOOK_LINK = 'https://docsend.com/view/29vcfdqa5aupkmfp
 
 export const MAX_VIEW_WIDTH_FOR_MOBILE_NAVBAR = 1150
 
+// Sentry
+export const SENTRY_DSN = 'https://528794315c61463db7d5181ebc1d51b9@o95148.ingest.sentry.io/210579'
```

Comment:
I was going to ask if this needed to be an ENV variable, and then remembered that this is front-end code and this value will be publicly exposed anyway.  I do like centralizing this value into a constant like this.